packages/plugin-network-breadcrumbs/network-breadcrumbs.js

```diff
@@ -120,11 +120,21 @@ const monkeyPatchFetch = () => {
 
   const oldFetch = win.fetch
   win.fetch = function fetch (...args) {
-    let [url, options] = args
+    const [urlOrRequest, options] = args
+
     let method = 'GET'
-    if (options && options.method) {
-      method = options.method
+    let url
+
+    if (typeof urlOrRequest === 'string') {
+      url = urlOrRequest
+      if (options && options.method) {
+        method = options.method
+      }
+    } else {
```

Comment:
this block is entered when `typeof urlOrRequest !== 'string'` – it is then assumed to be an object, but it could also be `null` or `undefined` at this point.

If you call `fetch(null)` it causes the following error to be reported to Bugsnag:

```
TypeError: urlOrRequest is null
    fetch network-breadcrumbs.js:134
    <anonymous> (index):20
```